aws/data_source_aws_route53_zone.go

```diff
@@ -56,6 +56,16 @@ func dataSourceAwsRoute53Zone() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Computed: true,
 			},
+			"linked_service_principal": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"linked_service_description": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Since this attribute is never used as an input argument (e.g. for filtering/searching), it should not be marked `Optional: true` 👍 

```suggestion
```